src/actions/uiStateActions.js

```diff
@@ -13,6 +13,14 @@ export function selectCurrentGuide(guide: Guide): Action {
   return { type: "SELECT_CURRENT_GUIDE", guide };
 }
 
+export function selectCurrentGuideByID(guideID: number): ThunkAction {
```

Comment:
we can remove this action later. it's just a bridge between the old ways and the new.